src/App.js

```diff
@@ -43,13 +44,21 @@ const PageRoute = ({ path, children }) => {
   )
 }
 
-const NavbarRoute = ({ path, children, name, handleLogout }) => {
+const AuthPageRoute = ({ path, children }) => {
+  const { isAuthed } = useAuth()
+  return <Route path={path}>{isAuthed ? <Page>{children}</Page> : <Redirect to="/login" />}</Route>
+}
+
+const NavbarRoute = ({ path, children }) => {
```

Comment:
Quick comment about `NavbarRoute`, since a user's name and the logout button is not displayed on every single screen, can you make it take in `name` and `handleLogout`? 

screens that display name and logout:
- /application/:part
- /application/review

logout only:
- /application/confirmation

no name. no logout:
- /login

Reply:
can we do this in a different PR? I think we should merge this PR Asap